samples/ai-image-tagging/src/lib/content-type.js

```diff
@@ -0,0 +1,37 @@
+
+const findField = (fieldSelectorFn, fieldNameIncludes) => (contentType) => {
+  const possibleFields = fieldSelectorFn(contentType);
+
+  if (possibleFields.length === 0) {
+    return null
+  } else if (possibleFields.length > 1) {
+    const topCandidates = possibleFields.filter(
```

Comment:
As we're not sorting the possible candidates and are only interested in one of them you could simplify this a little using [`array.find`](https://devdocs.io/javascript/global_objects/array/find)

```javascript
var topCandidate = possibleFields.find(field => ...)
```

Reply:
Deleted that part of the code for the migration to instance parameters